### DISCUSSION TITLE
XCT Next Release

### DISCUSSION THREAD

**Discussion by GUOLDEV** [xamarin/XamarinCommunityToolkit#1867] [General]
Any idea when the XCT next version will be released?

**Comment by bijington** [comment]
I would expect one in the next few days 😉

**Reply by GUOLDEV** [reply]
@bijington any update about the release?

**Comment by GUOLDEV** [comment]
@bijington @jfversluis @pictos 
I would love to keep helping XCT fixing bugs etc, but I would like to bring in a problem that I am facing, so maybe we have others in the same situation.

After fixing a bug in CameraView, it's taking too long for an official release, my clients cannot wait 2 months for the fix, so I will need to do something that I am doing everything to avoid since I started using XCT, I will need to copy it and implement locally into my code.

Don't get me wrong, I want to be part of the solution, not just the problem, so please let me know if I can help in any way 👍

What about giving some priority for hot fixes?

Or breaking down the XCT, so we can have small and regular releases?

**Reply by pictos** [reply]
@GUOLDEV I know that we took too long to release a new version of XCT, you're totally right on that. We're planning a release of all packages soon-sh. Meantime you can pack the XCT locally and use the NuGet that you generate on your machine and ship it or use our [nightly](https://pkgs.dev.azure.com/xamarin/public/_packaging/XamarinCommunityToolkitNightly/nuget/v3/index.json) feed ( I believe this is still working).

Again, I'm sorry for a long time waiting for a new release, we will do our best to not repeat this

**Reply by GUOLDEV** [reply]
Just saw that it was released 🙏
